bin/bb8/remote_paths.py

```diff
@@ -0,0 +1,38 @@
+import re
+from os.path import join
+
+
+class RemotePaths(object):
+    def __init__(self, target_name, starport):
+        self.host = "{user}@{addr}".format(**starport)
+        self.remote_bucket_path = self._remote_bucket_path(target_name,
+                                                           starport)
+
+    @property
+    def data(self):
+        return self.get_path("data")
+
+    @property
+    def meta(self):
+        return self.get_path("meta")
+
+    @property
+    def metadata_file(self):
+        return join(self.meta, "metadata.json")
+
+    @property
+    def rsync_path(self):
+        return "{host}:{path}".format(host=self.host, path=self.data)
+
+    def get_path(self, directory):
```

Comment:
Is this intended for external use too? If not should this be `_get_path`?

Reply:
Fixed